lib/gems/pending/util/miq_file_storage.rb

```diff
@@ -142,22 +142,8 @@ def upload_single(file) # rubocop:disable Lint/UnusedMethodArgument
 
     def upload_splits
       @position = 0
-      begin
-        until source_input.eof?
-          upload_single(next_split_filename)
-          @position += byte_count
-        end
-      rescue Errno::EIO
```

Comment:
This is going to go by bye because of a rebase, but want to leave a diff here in case it becomes relevant to bring it back in the future (I don't want to spend hours looking up how to fix this again):

```diff
commit 90ce1208de367f4c6b0119d772a9b411f7a7080d
Author: Nick LaMuro <nicklamuro@gmail.com>
Date:   Wed Aug 29 16:00:49 2018 -0500

    [FIXUP] Skip unneeded PTY based rescue
    
    Now that we aren't using PTY, this rescue should no longer be needed
    (hopefully).  Running this against Travis failed it without this last
    time, so if the specs still pass with this removed, I will rebase this
    in the the original commits.

diff --git a/lib/gems/pending/util/miq_file_storage.rb b/lib/gems/pending/util/miq_file_storage.rb
index dc7d488..43d7848 100644
--- a/lib/gems/pending/util/miq_file_storage.rb
+++ b/lib/gems/pending/util/miq_file_storage.rb
@@ -142,22 +142,8 @@ class MiqFileStorage
 
     def upload_splits
       @position = 0
-      begin
-        until source_input.eof?
-          upload_single(next_split_filename)
-          @position += byte_count
-        end
-      rescue Errno::EIO
-        # Error::EIO is raised when you read from a closed PTY on Linux.  See:
-        #
-        #   https://stackoverflow.com/a/10306782/3574689
-        #   https://ruby-doc.org/stdlib-2.2.3/libdoc/pty/rdoc/PTY.html#module-PTY-label-Example
-        #
-        # Or for the neckbeards:
-        #
-        #   $ ri PTY
-        #
-        # Done with reading at this point, but update @position... just incase
+      until source_input.eof?
+        upload_single(next_split_filename)
         @position += byte_count
       end
     end
```